guides/release/tutorial/7.md

```diff
@@ -0,0 +1,43 @@
+
+With Ember, we can do better than that! Instead of the plain-old `<a>` tag, Ember provides an alternative called `<LinkTo>`. For example, here is how you would use it on the contact page we just created:
+
+<!-- TODO: format diff -->
+```
+<div class="jumbo">
+  <div class="right tomster"></div>
+  <h2>Contact Us</h2>
+  <p>
+    Super Rentals Representatives would love to help you<br>
+    choose a destination or answer any questions you may have.
+  </p>
+  <address>
+    Super Rentals HQ
+    <p>
+      1212 Test Address Avenue<br>
+      Testington, OR 97233
+    </p>
+    <a href="tel:503.555.1212">+1 (503) 555-1212</a><br>
+    <a href="mailto:superrentalsrep@emberjs.com">superrentalsrep@emberjs.com</a>
+  </address>
+  <LinkTo @route="about" class="button">About</LinkTo>
+</div>
+```
+
+There is quite a bit going on here, so let's break it down.
+
+`<LinkTo>` is an example of a *[component](TODO: link to component)* in Ember &mdash; you can tell them apart from regular HTML tags because they start with an uppercase letter. Along with regular HTML tags, components are a key building block that we can use to build up an app's user interface.
+
+We have a lot more to say about components later, but for now, you can think of them as a way to provide *[custom tags](TODO: link to custom tags)* to supplement the built-in ones that came with the browser.
+
+The `@route=...` part is how we pass *[arguments](TODO: link to arguments)* into the component. Here, we use this to specify _which_ route we want to link to. Note that this should be the _name_ of the route, not the path, which is why we specified `"about"` instead of `"/about"`, and `"contact"` instead of `"/getting-in-touch"`.
+
+Components can also take the usual HTML attributes as arguments as well. In our example, we added a `"button"` class for styling purposes, but we could also specify other attributes as we see fit, such as the [ARIA](TODO: link to ARIA roles) `role` attribute. These are passed without the `@` symbol (`class=...` as opposed to `@Class=...`), so that Ember will know they are just regular HTML attributes.
+
+Under the hood, the `<LinkTo>` component generates a regular `<a>` tag for us with the appropriate `href` for the specific route. This allows for perfect interoperability for all screen readers, as well as the ability for our users to bookmark the link or open it in a new tab.
+
+However, when clicking on one of these special links, Ember will intercept the click, render the content for the new page, and update the URL &mdash; all performed locally without having to wait for the server, thus avoiding a full page refresh.
```

Comment:
```suggestion
However, when we click one of these special links, Ember will intercept the click, render the content for the new page, and update the URL &mdash; all performed in the browser without having to wait for the server, thus avoiding a full page refresh.
```

Reply:
I think technically there should be no spaces on either side of em dashes, but I guess it's fine as long as it's used consistently

Reply:
I'll defer to @vaidehijoshi

Reply:
I kept spaces in them because it was much easier to read/type in markdown/the code editor. I've been consistently using the spaces in the tutorial prose, so it should be easy for me to do a find-and-replace for it at the end.